HYF1.css

```diff
@@ -0,0 +1,289 @@
+/*navbar*/
```

Comment:
Better to name your files in lowercase as best practices says so.
You don't have to name your HYF
something like : index.html, index.css